test/base/intrinsics.cpp

```diff
@@ -2,6 +2,8 @@
 //
 // SPDX-License-Identifier: BSD-3-Clause
 
+/*@GKO_PREPROCESSOR_FILENAME_HELPER@*/
```

Comment:
Maybe I am missing something, wouldn't this be now after preprocessing be `/*/*#line 5 .... */*/` ?

Reply:
Okay, the above comment clarifies it. :)

Reply:
The actual string will be
```
/**/
#line 5 "file.cpp"
/**/